kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -367,25 +402,33 @@ define([
                     const rowFileName = data[2];
                     //use the name to look up all the data we have
                     let rowFileData = getFileFromName(rowFileName);
+                    
+                    //find the initial singly mapped datatype from the staging service
+                    let suggestedTypes = $(data[5]).find('optgroup[label="Suggested Types"]');
+                    let suggestedType = null;
+                    if (suggestedTypes.children().length == 1) {
+                        var option = suggestedTypes.find('option');
+                        suggestedType = { 'id': option.val(), 'title': option.html() };
+                    }
 
+                    //Get selected?
                     function changeImportButton(event) {
                         const checked = event.currentTarget.checked;
-
                         if (checked) {
                             stagingAreaViewer.enableImportButton();
                         } else {
                             /*
-                                check state of all checkboxes
-                                if any are checked we leave import button enabled
+                            check state of all checkboxes
+                            if any are checked we leave import button enabled
                             */
                             let anyCheckedBoxes = $('input.kb-staging-table-body__checkbox-input:checked');
```

Comment:
you could do this check above before the `if(checked)` statement, and turn that into `if(checked || anyCheckedboxes.length)`

Reply:
@leia-sefkin Sorry I don't understand. Which check? I didn't make any edits here